### PR TITLE
Rename because of feedback

### PR DESCRIPTION
**Pull request by davidfowl**

### CODE REVIEW

**Comment by akoeplinger** [comment]
There still two references to dotnet:
>Only the second of these can be converted to a dotnet5.x based reference, 

> PCL projects will be able to consume packages built for .NET Platform Standard (dotnet5.x) but not vice versa.

File: Documentation/project-docs/standard-platform.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ In general, class libraries which target a lower .NET Platform Standard version
 
 | Target Platform Name | Alias |  |  |  |  |  |
 | :---------- | :--------- |:--------- |:--------- |:--------- |:--------- |:--------- |
-|.NET Platform Standard | dotnet | 5.1 | 5.2 | 5.3 | 5.4 | 5.5 |
+|.NET Platform Standard | netstandard | 1.0 | 1.1 | 1.2 | 1.3 | 1.4 |
```
Comment (weshaggard) [inline Documentation/project-docs/standard-platform.md]: Did we consider making this shorter? Perhaps netstd?

Comment (davidsh) [inline Documentation/project-docs/standard-platform.md]: >Did we consider making this shorter? Perhaps netstd? 

I would vote we do not go with 'netstd' as a choice. The 'std' suffix might be misinterpreted.

What about 'netplat' (i.e. .NET Platform)?

Comment (davidfowl) [inline Documentation/project-docs/standard-platform.md]: yea std didn't go over well. A few more characters wont hurt and it's clear what it represents.

**Pull request merged**